{{cookiecutter.folder_name}}/main.tex

```diff
@@ -12,7 +12,8 @@
 % You could also define the document class directly
 %\documentclass[]{emulateapj}
 
-\input{macros}
+% Custom commands from LSST DESC, see texmf/styles/lsstdesc_macros.sty
```

Comment:
I added an extra comment, so people know where to go looking for these macros. We should probably add a note in the README that files in the texmf folder shoudl not be edited by the user (because they will be over-written during a `make update`)